File: barber-flow-api/Barber.Flow.Api/Barber.Flow.Domain/Entities/BarberShop.cs
namespace Barber.Flow.Domain.Entities;

public class BarberShop
{
    public string Id { get; set; } = string.Empty; // SHOP-0001

    public string Name { get; set; } = string.Empty;

    public string? Phone { get; set; }

    public string? Address { get; set; }

    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;

    public DateTime UpdatedAt { get; set; } = DateTime.UtcNow;

    public string CreatedBy { get; set; } = string.Empty;

    public string UpdatedBy { get; set; } = string.Empty;
}

File: barber-flow-mobile/Barber.Flow.Mobile/src/types/barberShop.ts
export interface BarberShop {
    id: string;
    name: string;
    phone?: string;
    address?: string;
    createdAt: string;
    updatedAt: string;
}

export interface BarberShopDraft {
    name: string;
    phone?: string;
    address?: string;
}
